### PR TITLE
Add missing links, fix some text bugs and add elsif usage in macro

### PR DESCRIPTION
**Pull request by Heaven31415**
For someone new it might not be so obvious that you can also use **elsif** in macros, that's why I modified existing example to show how to use it.

### CODE REVIEW

File: installation/from_source_repository.md [hunk 1]
```diff
@@ -14,10 +14,10 @@ If you want to contribute then you might want to install Crystal from sources.
...
 ```
 
-5. Run `make` to build your own version of the compiler
+5. Run `make` to build your own version of the compiler.
 6. Run `make spec` to ensure all specs pass, and you've installed everything correctly.
-7. Use `bin/crystal` to run your crystal files
+7. Use `bin/crystal` to run your crystal files.
 
-If you would like more information about the new `bin/crystal`, check out the [using the compiler](https://crystal-lang.org/docs/using_the_compiler/) documentation.
+If you would like more information about the new `bin/crystal`, check out [using the compiler](https://crystal-lang.org/docs/using_the_compiler/) documentation.
```
Comment (RX14) [inline installation/from_source_repository.md]: `bin/crystal` is hardly new any more, we should probably just leave out the "the new".

Comment (RX14) [inline installation/from_source_repository.md]: This sounds better with the "the" as before.

File: installation/from_source_repository.md [hunk 2]
```diff
@@ -14,10 +14,10 @@ If you want to contribute then you might want to install Crystal from sources.
...
 
-Note: The actual binary is built in to `.build/crystal`, but the `bin/crystal` wrapper script is what you should use to run crystal.
+Note: The actual binary is built into `.build/crystal`, but the `bin/crystal` wrapper script is what you should use to run crystal.
```
Comment (RX14) [inline installation/from_source_repository.md]: "into" is wrong here, we're just describing where the file is placed: "The actual binary is placed at ..."

**Comment by Heaven31415** [comment]
Okay, I fixed my bug and taken your suggestions into consideration. Can I leave this PR as it is or maybe should I squash those commits into one or something? (not that experienced with PRs).

**Comment by mverzilli** [comment]
Thank you @Heaven31415!

**Pull request merged**